docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,29 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
+
+    - In the benchmarking pod's YAML file, ``conf/helmfile.d/0410.benchmarking.yaml``, the ``MODEL`` environmental variable has been set to the exact model name and version we used in our benchmarking, ``NuclearSegmentation:2``. The benchmarking cluster, when you create it, will look for the model files corresponding to this name and version in the ``models/`` folder of your benchmarking bucket. To recreate our benchmarking data, please copy the ``models/NuclearSegmentation`` folder in the Van Valen Lab's `kiosk-benchmarking bucket <https://console.cloud.google.com/storage/browser/kiosk-benchmarking>`_ to the same location in your bucket.
+    - Also in the benchmarking pod's YAML file is an environmental variable, ``FILE``, which is set to ``zip100.zip``. The benchmarking pod will look for a file by this name in the bucket you specified above at the location ``uploads/[file_name]``. We have a file named ``zip100.zip`` located in our ``kiosk-bencharking`` bucket at ``sample_data/zip100.zip``. This file consists of 100 microscopy images, which we used as the basis for all our benchmarking runs. To recreate our benchmarking data, please copy ``sample_data/zip100.zip`` from the Van Valen Lab's `kiosk-benchmarking bucket <https://console.cloud.google.com/storage/browser/kiosk-benchmarking>`_ to ``uploads/zip100.zip`` in your bucket.
+    - A final variable in the benchmarking pod's YAML file is ``COUNT``, which determines how many times the ``zip100.zip`` file will be submitted to the cluster for processing. This controls the total number of images the cluster will process over the course of the benchmarking run. For example, if you want to do a 10,000-image benchmarking run, you would set ``COUNT`` TO 100, since 100*100 = 10,000. Benchmarking data was presented in the paper for 10,000-image, 100,000-image, and 1,000,000-image runs. Accordingly, depending on which subset of the paper's benchmarking data the user wishes to recreate, ``COUNT`` should be set to either 100, 1,000, or 10,000.
+
+3. Deploy a DeepCell Kiosk as you normally would. While navigating the cluster configuration menu, pay special attention to two configuration settings:
+    - The bucket name you provide should be that of the benchmarking bucket from step 1. To ensure the benchmarking process works properly, this bucket should not be attached to any other cluster you might have running.
+    - The Maximum Number of GPUs is relevant to the benchmarking process. In the paper, benchmarking data was presented for clusters with maxima of 1, 4, and 8 GPUs. Choose the appropriate maximum for the benchmarking dataset you would like to recreate.
```

Comment:
I think this doc should have less a focus on our own benchmarking run, and more of a focus on how they can use the utility for their own benchmarking runs (which may or may not include replicating our own).

Reply:
Ok, in all other instances, I included an explicit statement that the user could choose whatever value they desired in addition to telling them what settings they'd need to pick to recreate the paper figure. What do you think about that approach?